src/rfq/protocols/bebop/state.rs

```diff
@@ -13,8 +14,8 @@ use tycho_common::{
 
 #[derive(Debug)]
 pub struct BebopState {
-    pub base_token: String,
-    pub quote_token: String,
+    pub base_token: Token,
+    pub quote_token: Token,
     pub last_update_ts: u64,
     pub bids: Vec<(f64, f64)>,
     pub asks: Vec<(f64, f64)>,
```

Comment:
Still need to change this to match your PR. Should be a short change. Pls continue with the assumption that I've changed it!